src/blocks/table/table.ts

```diff
@@ -0,0 +1,63 @@
+import { html, render } from 'lit';
+
+interface TableCell {
+  text: string;
+  styles?: string;
+}
+
+interface TemplateArgs {
+  headers: TableCell[];
+  data: TableCell[][];
+}
+
+const template = (args: TemplateArgs) => {
+  const { headers, data } = args;
+  return html`
+    <div class="table-wrapper">
+      <table>
+        <thead>
+          <tr>
+            ${headers.map((header) => html`<th style="${header.styles || ''}">${header.text}</th>`)}
+          </tr>
+        </thead>
+        <tbody>
+          ${data.map(
+            (row) => html`
+              <tr>
+                ${row.map((cell) => html`<td style="${cell.styles || ''}">${cell.text}</td>`)}
```

Comment:
hier bitte prüfen wenn man redaktionell strong italic usw. in einem table cell verwendet auch gerendert werden...